docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+# AMWA BCP-XXX-YY: EDID Mapping in NMOS Systems \[Work In Progress\]
+
+
+## Background
+
+AMWA NMOS specifications provide resources for specifying receiver capabilities of an NMOS device.  Recent extension of these resources allow NMOS Controllers developed for PRO A/V User-Stories to be created.  In particular, these NMOS Controllers include Controllers developed for the AIMS sponsered IPMX set of open standards [IPMX](#ipmx-ref). 
```

Comment:
>Recent extension

It's not clear what extension is meant.

> NMOS Controllers

It looks like a brand new term which meaning is unclear.

Reply:
This paragraph might be a better fit in another document